src/test/ui/error-codes/E0225.stderr

```diff
@@ -1,9 +1,22 @@
 error[E0225]: only auto traits can be used as additional traits in a trait object
-  --> $DIR/E0225.rs:2:32
+  --> $DIR/E0225.rs:6:32
    |
 LL |     let _: Box<std::io::Read + std::io::Write>;
-   |                                ^^^^^^^^^^^^^^ non-auto additional trait
+   |                -------------   ^^^^^^^^^^^^^^ additional non-auto trait
+   |                |
+   |                first non-auto trait
 
-error: aborting due to previous error
+error[E0225]: only auto traits can be used as additional traits in a trait object
+  --> $DIR/E0225.rs:8:16
+   |
+LL | trait Foo = std::io::Read + std::io::Write;
+   |             -------------   -------------- additional non-auto trait
+   |             |
+   |             first non-auto trait
+...
+LL |     let _: Box<Foo>;
+   |                ^^^
```

Comment:
give this span a label like "trait alias used here to create a trait object"

(also I think this ui file needs updating)